core/modes/language_modes.py

```diff
@@ -9,18 +9,17 @@
     "batch": "bat",
     "c": "c h",
     # 'cmake': 'cmake',
-    # 'cplusplus': 'cpp hpp',
+    # "cpp": "cpp hpp",
```

Comment:
Not sure why you are renaming `cplusplus` to `cpp`?

Reply:
That is the default abbreviation.

Reply:
I'm concerned it might break existing people with modes (e.g. #777).

Reply:
Yes since it wasn't even active I kind of feel this is quite harmless. I will leave that decision to you. If you prefer I can revert

Reply:
@knausj85 any opinion?

Reply:
This is what we match on with code.language, yes? cplusplus feels more clear in that case.

Reply:
Correct
Okay I have reverted to the previous identifier.